pkg/services/sqlstore/migrations/live.go

```diff
@@ -0,0 +1,24 @@
+package migrations
+
+import (
+	. "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+)
+
+func addLiveMigrations(mg *Migrator) {
+	liveChannel := Table{
+		Name: "live_channel",
+		Columns: []*Column{
+			{Name: "id", Type: DB_BigInt, Nullable: false, IsPrimaryKey: true, IsAutoIncrement: true},
+			{Name: "org_id", Type: DB_BigInt, Nullable: false},
+			{Name: "channel", Type: DB_NVarchar, Length: 189, Nullable: false},
+			{Name: "data", Type: DB_Text, Nullable: false},
+			{Name: "created", Type: DB_DateTime, Nullable: false},
```

Comment:
Looks like if table will also be used for configuration `data` column here can be nullable.